packages/core/src/v3/schemas/common.ts

```diff
@@ -77,7 +130,11 @@ export const TaskRun = z.object({
   tags: z.array(z.string()),
   isTest: z.boolean().default(false),
   createdAt: z.coerce.date(),
+  startedAt: z.coerce.date(),
```

Comment:
Is this used in the `.list()` SDK call? Isn't the date null for some runs if they've not started?